src/test/codewhisperer/tracker/codewhispererCodeCoverageTracker.test.ts

```diff
@@ -62,18 +191,20 @@ describe('codewhispererCodecoverageTracker', function () {
             resetCodeWhispererGlobalVariables()
             CodeWhispererCodeCoverageTracker.instances.delete(language)
         })
-
+        afterEach(function () {
+            sinon.restore()
+        })
         it(' emits codecoverage telemetry ', function () {
-            const sample = "print('Hello World!)"
             const tracker = CodeWhispererCodeCoverageTracker.getTracker(language, mockGlobalStorage)
             const assertTelemetry = assertTelemetryCurried('codewhisperer_codePercentage')
-            tracker.setAcceptedTokens(sample)
-            tracker.emitCodeWhispererCodeContribution()
+            tracker?.addAcceptedTokens(`test.py`, { range: new vscode.Range(0, 0, 0, 7), text: `print()`, accepted: 7 })
```

Comment:
Exposing `addAcceptedTokens` feels clunky. Isn't `accepted` always a computed field of `text`? And isn't `text` always the text document text provided by `range`? So a 'token' can be computed using a text document + a range.

Reply:
Field `accepted`  is used to store the number of unmodified accepted tokens

Reply:
Right I think the clunky-ness comes from lack of explicit initialization of the tracking behavior. Not a blocker.